sounds/models.py

```diff
@@ -1795,9 +1795,10 @@ def get_attribution(self):
         users = User.objects.filter(sounds__in=sounds_list).distinct()
         # Generate text file with license info
         licenses = License.objects.filter(sound__pack=self).distinct()
-        attribution = render_to_string("sounds/pack_attribution.txt",
-            dict(users=users,
-                pack=self,
+        attribution = render_to_string("sounds/multiple_sounds_attribution.txt",
+            dict(type=self.__class__.__name__,
```

Comment:
wow, this is ugly :S 
just hardcode the name, e.g. `type="pack"`, it is clearer.